docs/language/blocks-and-lambdas.mdx

```diff
@@ -16,13 +27,44 @@ main:
 ```
 
 Here we used the automatic block argument, `it`, [see
-below](#block-arguments)
+below](#block-arguments).
 
 Syntactically `repeat` and `do` look like they are built-in to the language
 like `for` and `if` are, but they are actually normal methods that use the
 block feature.
 
-## Basic use of blocks
+### Lambdas
+
+Lambdas are more flexible function-like objects introduced with a double colon `::`.
+They can be stored in variables, passed as arguments, and returned from functions.
+While not as efficient as blocks, lambdas offer greater versatility.
+
+Example of a lambda:
+
+```toit
+printer/Lambda? := null
```

Comment:
Why is this a top-level variable?

Reply:
I was trying to show that it can be stored in places where a block can't be.
Changed to `printer = create-printer`.